### PR TITLE
Cloud Bigtable: helloWorld sample

### PR DESCRIPTION
**Pull request by elisheva-qlogic**
Fixes #<issue_number_goes_here> (it's a good idea to open an issue first for context and/or discussion)

### CODE REVIEW

File: google-cloud-examples/src/test/java/com/google/cloud/examples/bigtable/ITHelloWorld.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  }
+
+  @Test
+  public void testWriteToAndReadFromTable() {
+    // Write to table
+    helloWorld.writeToTable();
+    Row row = dataClient.readRow(tableId, "rowKey0");
+    assertNotNull(row);
+  }
+
```
Comment (igorbernstein2) [inline google-cloud-examples/src/test/java/com/google/cloud/examples/bigtable/ITHelloWorld.java]: Please either add the missing tests or add TODOs for:
- readSingleRow
- readTable

Separately, add a test `testRunDoesNotFail` that calls run() on HelloWorld to ensure that the run as a whole works

File: google-cloud-examples/src/test/java/com/google/cloud/examples/bigtable/ITHelloWorld.java [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  private String generateTableId() {
+    return String.format(
+        "%s-%016x-%d", TABLE_PREFIX, System.currentTimeMillis(), new Random().nextLong());
+  }
+
+  private static void garbageCollect() {
+    Pattern timestampPattern = Pattern.compile(TABLE_PREFIX + "-([0-9]+)");
```
Comment (igorbernstein2) [inline google-cloud-examples/src/test/java/com/google/cloud/examples/bigtable/ITHelloWorld.java]: This doesnt match the generateTableId format above. The correct format is:
TABLE_PREFIX + "-([0-9a-f]+)-([0-9a-f]+)"

File: google-cloud-examples/src/test/java/com/google/cloud/examples/bigtable/ITHelloWorld.java [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+  public void testWriteToAndReadFromTable() {
+    // Write to table
+    helloWorld.writeToTable();
+    Row row = dataClient.readRow(tableId, "rowKey0");
+    assertNotNull(row);
+  }
+
+  private String generateTableId() {
+    return String.format(
+        "%s-%016x-%d", TABLE_PREFIX, System.currentTimeMillis(), new Random().nextLong());
```
Comment (igorbernstein2) [inline google-cloud-examples/src/test/java/com/google/cloud/examples/bigtable/ITHelloWorld.java]: please use hexadecimal for the random segment:
%s-%016x-%x

File: google-cloud-examples/src/test/java/com/google/cloud/examples/bigtable/ITHelloWorld.java [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+  private static void garbageCollect() {
+    Pattern timestampPattern = Pattern.compile(TABLE_PREFIX + "-([0-9]+)");
+    for (String tableId : adminClient.listTables()) {
+      Matcher matcher = timestampPattern.matcher(tableId);
+      if (!matcher.matches()) {
+        continue;
+      }
+      String timestampStr = matcher.group(1);
+      long timestamp = Long.parseLong(timestampStr);
```
Comment (igorbernstein2) [inline google-cloud-examples/src/test/java/com/google/cloud/examples/bigtable/ITHelloWorld.java]: timestampStr is in hexadecimal format. Please add a base of 16 to parseLong

**Comment by chingor13** [comment]
We'll want add a new kokoro config in .kokoro/presubmit (and continuous and nightly) that looks like `bigtable-it.cfg`, but for the samples. The INTEGRATION_TEST_ARGS will be a place where you can configure any system properties necessary for the test to run.

We'll separately need to update our internal config to enable the new jobs.

**Comment by igorbernstein2** [comment]
@chingor13 If this looks good to you, we can merge it

**Pull request merged**